scripter/lua/lua.go

```diff
@@ -101,3 +99,23 @@ func (l *luaScripter) GetConnection(service string, conn net.Conn) scripter.Conn
 	return &scripter.ConnectionStruct{Service: service, MyConn: sConn}
 }
 
+// Check whether scripter can handle incoming connection for the peeked message
+// Returns true if there is one script able to handle the connection
+func (l *luaScripter) CanHandle(service string, pMessage string) bool {
+	for _, ls := range l.canHandleStates[service] {
+		canHandle, err := callCanHandle(ls, pMessage)
```

Comment:
pMessage is called message everywhere else, maybe use it here as 'message' as well?